RiotSharp/RiotApi.cs

```diff
@@ -214,15 +214,17 @@ public Summoner GetSummonerByName(Region region, string summonerName)
         #region Champion
         public List<Champion> GetChampions(Region region, bool freeToPlay = false)
         {
+            var freeToPlayString = freeToPlay ? "true" : "false";
             var json = requester.CreateGetRequest(PlatformRootUrl + ChampionsUrl, region,
-                new List<string> { string.Format("freeToPlay={0}", freeToPlay ? "true" : "false") });
+                new List<string> { $"freeToPlay={freeToPlayString}" });
```

Comment:
This returns `True` or `False` instead of `true` and `false` and I know that Riot's i boolean parameter interpretation is casesensitive. Use `freeToPlay.ToString().ToLower()` instead.